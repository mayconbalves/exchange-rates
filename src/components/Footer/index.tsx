import styles from './styles.module.css'
const Footer = () => {
  return (
    <footer className={styles.wrapper}>
      <img src="./logo.png" alt="B3 logo" />
    </footer>
  )
}

export default Footer
