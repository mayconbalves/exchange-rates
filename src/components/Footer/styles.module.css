.wrapper {
  background-color: #000000;
  bottom: 0;
  display: flex;
  justify-content: center;
  height: 72px;
  position: fixed;
  width: 100%;
}
