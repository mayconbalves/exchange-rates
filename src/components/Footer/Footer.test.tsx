import { render, screen } from '@testing-library/react'
import Footer from '.'

describe('Footer Component', () => {
  it('Should be render component', () => {
    render(<Footer />)
    const footerElement = screen.getByRole('contentinfo')
    expect(footerElement).toBeInTheDocument()
  })

  it('Should be render image logo', () => {
    render(<Footer />)
    const imageElement = screen.getByRole('img', { name: /b3 logo/i })
    expect(imageElement).toBeInTheDocument()
    expect(imageElement).toHaveAttribute('src', './logo.png')
  })

  it('Should be create snapshot', () => {
    const { container } = render(<Footer />)

    expect(container).toMatchSnapshot()
  })
})
