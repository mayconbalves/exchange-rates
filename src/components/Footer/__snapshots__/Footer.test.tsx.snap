// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Should be create snapshot 1`] = `
<div>
  <footer
    class="wrapper"
  >
    <img
      alt="B3 logo"
      src="./logo.png"
    />
  </footer>
</div>
`;
